### ISSUE TITLE
Implement basic accessibility features (keyboard navigation) for the Results pane

### ISSUE THREAD

**Issue by goplayerjuggler** [mtxr/vscode-sqltools#687]
**Is your feature request related to a problem? Please describe.**
It should be possible to navigate the results pane using normal keyboard actions like tab, the arrow keys (optionally using Shift or Control for multi select of cells) and then Control C to copy the contents of the cell(s), _without using the mouse_.

**Describe the solution you'd like**
It should be possible to use Tab (or some other key, maybe F6 like in SQL studio) to toggle the focus between each grid in the result view, followed by the text displayed in the Messages section.

**Describe alternatives you've considered**
It's possible to select the info using the mouse - but having an alternative to using the mouse would be nice!

**Comment by gimbo** [comment]
Yes please! At the moment the "Run This File" command is very unpleasant to use because every query gets its own "sub tab", the first one is always the open one (I usually care more about the last one as that's what I'm working on), and there's no way to navigate except with the mouse. This makes iterating on an entire SQL file (e.g. one which starts with teardown/setup, followed by queries) much fiddlier than it needs to be; particularly combined with #632 and #635, for me this makes the results-viewing part of this extension the part which just doesn't work well — whereas everything else is rather good!
